docs/hacking-an-agent.md

```diff
@@ -1,99 +1,222 @@
 ## Preliminaries
 
-These instructions detail the Python code you need for running an AEA outside the `cli` tool.
+These instructions detail the Python code you need for running an AEA outside the `cli` tool, using the code interface. 
 
-<div class="admonition note">
-  <p class="admonition-title">Note</p>
-  <p>You have already coded up your agent. See the <a href="../aea/skill-guide/" target=_blank>build your own skill guide</a> for a reminder.</p>
-</div>
+  
+This guide assumes you have already followed the [quick start](quickstart.md) guide and so have the framework installed and the packages and scripts directory downloaded into the directory you are working in.
```

Comment:
I would say they need to specifically follow the `Preliminaries` and `Installation` sections of the quickstart.

Reply:
agreed